_posts/2021-04-27-a-gentle-introduction-to-iceberg.md

```diff
@@ -0,0 +1,276 @@
+---
+layout: post
+title:  "Trino On Ice I: A Gentle Introduction To Iceberg"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+canonical_url: https://blog.starburstdata.com/trino-on-ice-i-a-gentle-introduction-to-iceberg
```

Comment:
This generates the following html, which [is necessary for SEO when syndicating content](https://www.shoutmeloud.com/relcanonical-wordpresss-content-syndication-seo.html)

```
<link rel="canonical" href="https://blog.starburstdata.com/trino-on-ice-i-a-gentle-introduction-to-iceberg">
```